client/components/SignUp.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { Redirect } from 'react-router-dom';
+
+function SignUp(props) {
+  const [toHome, setToHome] = useState(false);
+  const { register, handleSubmit, reset } = useForm();
+
+  const { create } = props;
+
+  const onSubmit = (data) => {
+    const userData = JSON.stringify(data);
+    const userForm = JSON.parse(userData);
+    create(userForm);
+    reset({});
+    // setTimeout(() => setToHome(true), 2000);
+    setToHome(true);
```

Comment:
Not sure it makes much of a difference, but you could just redirect to '/' here at the end of onSubmit rather than using a toHome state + ternary operator on line 22